strscan.gemspec

```diff
@@ -17,12 +17,20 @@ Gem::Specification.new do |s|
   s.description = "Provides lexical scanning operations on a String."
 
   s.require_path = %w{lib}
-  s.files = %w{ext/strscan/extconf.rb ext/strscan/strscan.c}
-  s.extensions = %w{ext/strscan/extconf.rb}
+
+  jruby = true if Gem::Platform.new('java') =~ s.platform or RUBY_ENGINE == 'jruby'
```

Comment:
Is this necessary or just `RUBY_ENGINE == 'jruby'` would be enough?

Reply:
@headius What do you think about this?

Reply:
https://github.com/ruby/strscan/pull/36